scripts/astrokat-uvcoverage.py

```diff
@@ -1,13 +1,16 @@
-#!/usr/bin/env python
-"""Theoretical UV coverage from antenna position"""
+#! usr/bin/env python
+"""Theoretical UV coverage from antenna positions"""
 
 from __future__ import print_function
 from astropy import units as u
 from astropy.coordinates import Longitude, Latitude, EarthLocation
+from astropy.coordinates import ICRS, SkyCoord
+from datetime import datetime, timedelta
 from numpy import recarray
 
 import argparse
 import astropy.constants as phys
+import ephem
```

Comment:
OK, but I thought this script was trying to do everything with astropy?

Reply:
astropy does not have an "observer" for that you have to use astroplan, which in itself is a very good tool, but it is >py3.5, so I had to use ephem for the interim until 2.7 is fully retired
Since ephem is only used for the observer, but the targets are astropy, I will sidestep the rounding issues of ephem in pointing